app/views/telephone_appointments/new.html.erb

```diff
@@ -1,18 +1,22 @@
 <div class="l-grid-row">
-  <div class="l-column-two-thirds">
+  <div class="l-column-full">
     <h1>Book a phone appointment</h1>
 
-    <% if @telephone_appointment.errors.any? %>
+    <% if @telephone_appointment.errors.any? || (params[:continue] && @telephone_appointment.start_at.nil?) %>
       <div class="error-summary t-errors" role="group" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
         <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
           Unable to submit the form
         </h2>
+
         <p>Check the following:</p>
 
         <ul class="error-summary-list">
           <% @telephone_appointment.errors.each do |key, message| %>
             <li><%= link_to "#{TelephoneAppointment.human_attribute_name key} – #{message}", "#telephone_appointment_#{key}" %></li>
           <% end %>
+          <% if @telephone_appointment.start_at.nil? %>
```

Comment:
I think it's clearer to switch this around and state `unless @telephone_appointment.start_at?` here.